lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -460,12 +461,23 @@ protected function _getCreateTableSQL($tableName, array $columns, array $options
     }
 
     /**
+     * Tells whether a field type supports declaration of a default value.
+     *
+     * MySQL (as of 5.7.19) does not support default values for Blob and Text
+     * columns while MariaDB 10.2.1 does.
+     */
+    protected function isDefaultValueSupportedForType(Type $field) : bool
+    {
+        return !($field instanceof TextType || $field instanceof BlobType);
+    }
+
+    /**
      * {@inheritdoc}
      */
     public function getDefaultValueDeclarationSQL($field)
     {
-        // Unset the default value if the given field definition does not allow default values.
-        if ($field['type'] instanceof TextType || $field['type'] instanceof BlobType) {
+        // Unset the default value if the given field type does not allow default values.
+        if (!$this->isDefaultValueSupportedForType($field['type'])) {
```

Comment:
Space is required around `!`.

Reply:
Thanks. Sorry I messed with cs from the beginning.